.github/workflows/aws-deploy.yml

```diff
@@ -44,17 +48,11 @@ jobs:
         uses: youyo/aws-cdk-github-actions@v2
         with:
           cdk_subcommand: 'synth'
-          working_dir: 'cdk'
-        env:
-          DOCKER_IMAGE: ${{ env.IMAGE_PATH_AND_TAG }}
- 
+          cdk_args: '--context env=dev --debug'
+
       - name: cdk deploy
         uses: youyo/aws-cdk-github-actions@v2
         with:
           cdk_subcommand: 'deploy'
-          cdk_args: '--require-approval never'
-          working_dir: 'cdk'
+          cdk_args: '--require-approval never --context env=dev --progress events --debug'
```

Comment:
to specify `env=prod` would we create a branch within this repo' containing a change to this line?

Reply:
there are multiple options to handle that depending on the dev workflow that we want to use.  setting up a  dev/stage/prod branch is one option.